src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+// import classNames from 'classnames';
+// import React, { useState } from 'react';
```

Comment:
remove comments
but make this `import classNames from 'classnames';` active and use the library